README.en.md

```
@@ -49,13 +49,19 @@
 
 ## 📝 Project Description
 
-> [!NOTE]  
-> This is an open-source project developed based on [One API](https://github.com/songquanpeng/one-api)
+> [!NOTE]\
+> This is an open-source project developed based on
+> [One API](https://github.com/songquanpeng/one-api)
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)